cranelift/wasm/src/code_translator.rs

```diff
@@ -2193,14 +2229,17 @@ fn translate_unreachable_operator<FE: FuncEnvironment + ?Sized>(
 /// generate necessary IR to validate that the heap address is correctly
 /// in-bounds, and various parameters are returned describing the valid *native*
 /// heap address if execution reaches that point.
-fn prepare_addr<FE: FuncEnvironment + ?Sized>(
+fn prepare_addr<FE>(
```

Comment:
Can we document here what a `None` return value means? (I also looked in `bounds_checks` and `prepare_addr` doesn't seem to say either.) Does it mean that the access always traps (as one might infer from the "set unreachable and return" above)? Or something else?

Reply:
Yeah, it means the access always traps. Will add comments.